5-Heroku/server/db/dbController.js

```diff
@@ -0,0 +1,73 @@
+require('dotenv').config('../../../.env')
+const mongoose = require('mongoose')
+const itemModel = require('../models/item')
+const url = process.env.MONGO_URL;
+
+module.exports.connectToDB = async function () {
+    try {
+        await mongoose.connect(url, { useNewUrlParser: true })
+    } catch (err) {
+        throw err
+    }
+}
+module.exports.getAllItemsFromDB = async function (userID) {
+    try {
+        const itemList = await itemModel.find({ userID: userID }).lean(true)
+        itemList.forEach(item => item._id = item._id.toString())
+        return itemList
+    } catch (err) {
+        throw err
+    }
+}
+module.exports.getItemFromDB = async function (_id) {
+    try {
+        const items = await itemModel.find({ _id }).lean(true)
+        if (items.length) {
```

Comment:
why do u need .toString on id?

Reply:
the _id property in of type ObjectID so .toString saves me the trouble of using mixed types